docs/sources/administration/configuration.md

```diff
@@ -1201,18 +1201,22 @@ Configure Grafana's Jaeger client for distributed tracing.
 You can also use the standard `JAEGER_*` environment variables to configure
 Jaeger. See the table at the end of https://www.jaegertracing.io/docs/1.16/client-features/
 for the full list. Environment variables will override any settings provided here.
+These environment variables follow the `GF_<section>` convention above,
```

Comment:
Based on information about standard jaeger environment variables above (documented at https://www.jaegertracing.io/docs/1.16/client-features/) this needs to explain that you can use `GF_<section>` convention if you prefer as well.

The reason for using standard jaeger environment variables is because some uses that for all their services, including  Grafana, as convenience.